CHANGELOG.md

```diff
@@ -1,4 +1,20 @@
 # Changelog
+## [2.0.9](https://github.com/bakdata/kpops/releases/tag/2.0.9) - Release Date: [2023-09-19]
```

Comment:
rather than merging `main` into this branch, shouldn't we merge `main` into `v3`? I believe those changes shouldn't show up in the diff here

Reply:
Fixed :)